Example/src/SharedElementTransition/Card.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { View, Image, TouchableNativeFeedback } from 'react-native';
+import { ParamListBase } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { StackScreenProps } from '@react-navigation/stack';
+import Animated from 'react-native-reanimated';
+
+const photo = require('./assets/image.jpg');
+const Stack = createNativeStackNavigator();
+const AnimatedImage = Animated.createAnimatedComponent(Image);
```

Comment:
You can use `Animated.Image` directly:

https://github.com/software-mansion/react-native-reanimated/blob/073057f785346f1cdaa88a19680ae84f54a310a2/src/Animated.ts#L10